extensions/wikia/CommunityPage/styles/components/_insights-module.scss

```diff
@@ -1,77 +1,86 @@
 @import 'skins/shared/color';
-@import 'skins/oasis/css/core/breakpoints-variables';
 @import 'skins/shared/mixins/flexbox';
 
-$item-icon-size: 34px;
-$item-icon-margin-right: 6px;
-$item-data-width: calc(70% - #{$item-icon-size + $item-icon-margin-right});
-$item-link-width: 30%;
+$content-spacing: 18px;
+$content-spacing-large: 24px;
+$element-size: 48px;
+$font-size-small: 12px;
+$font-size-regular: 14px;
+$icon-size: 18px;
 
-$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen.svg';
+.community-page-insights-module {
+	@include flexbox();
+	@include flex-direction(column);
 
-@if $is-dark-wiki {
-	$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen-dark.svg';
-}
+	background-color: $color-page;
+	border: 1px solid $color-page-border-subtle;
+	margin-bottom: $content-spacing;
+	width: 239px;
 
-.community-page-insights-module {
-	margin-bottom: 34px;
+	.community-page-insights-module-article-data {
+		color: $color-links;
+	}
 
-	.community-page-insights-module-header {
-		font-size: 24px;
-		font-weight: bold;
-		line-height: 28px;
-		margin: 0 0 12px;
+	.community-page-insights-module-footer {
+		@include flexbox();
 	}
 
-	.community-page-insights-module-list {
-		list-style: none;
-		margin: 0;
+	.community-page-insights-module-full-list-link {
+		@include flex-grow(1);
+
+		color: $color-buttons;
+		display: block;
+		font-size: $font-size-small;
+		font-weight: bold;
+		line-height: $element-size;
+		margin-right: $content-spacing-large;
+		text-align: right;
+		text-transform: uppercase;
 	}
 
-	.community-page-insights-module-list-item {
+	.community-page-insights-module-header {
+		@include align-items(center);
 		@include flexbox();
 
-		align-items: center;
-		border-bottom: 1px solid $color-page-border-subtle;
-		padding: 20px;
+		background: $color-buttons;
+		color: $color-button-text;
+		font-size: $font-size-regular;
+		font-weight: bold;
+		line-height: 16px;
+		margin: 0 0 $content-spacing-large;
+		padding: $content-spacing;
+		text-transform: uppercase;
 	}
 
-	.community-page-insights-module-edit-icon {
-		background: url($edit-icon) no-repeat;
-		height: $item-icon-size;
-		margin-right: $item-icon-margin-right;
-		width: 24px;
+	.community-page-insights-module-header-icon {
+		fill: $color-buttons;
+		margin-right: 12px;
 	}
 
-	.community-page-insights-module-article-data {
-		color: $color-links;
-		overflow: hidden;
-		text-overflow: ellipsis;
-		white-space: nowrap;
-		width: $item-data-width;
+	.community-page-insights-module-help {
+		height: $element-size;
+		width: $element-size;
 	}
 
-	.community-page-insights-module-metadata {
-		color: $color-alternate-text;
-		font-size: 12px;
-
-		.communitypage-user-link {
-			display: inline-block;
-			max-width: 140px;
-			overflow: hidden;
-			text-overflow: ellipsis;
-			vertical-align: bottom;
-		}
+	.community-page-insights-module-help-icon {
+		fill: $color-page-border-subtle;
+		height: $icon-size;
+		margin: 15px;
```

Comment:
Not fully sure about whether this is similar case to L49 - I leave the call on whether to extract this one at your discretion